modules/ROOT/pages/configure-ingress-http-resource.adoc

```diff
@@ -0,0 +1,226 @@
+= Configuring Ingress with HTTP Route Template Custom Resource In Runtime Fabric
+
+Configure ingress support for Mule apps on Anypoint Runtime Fabric platform through the HTTP route template custom resource.
+
+== Benefits of Ingress Configuration with HTTP Route Template Custom Resource
+
+Current ingress support for Mule apps on the Runtime Fabric platform is confined to https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes ingress] resources, which restricts you from leveraging cloud vendor-native HTTP routing solutions, such as OpenShift Routes on OpenShift clusters, Istio or Service mesh. To address this behavior, Runtime Fabric supports a custom resource `httproutetemplates.rtf.mulesoft.com` template.
+
+Use the HTTP route template custom resource in Runtime Fabric to define and implement custom routing configurations to take advantage of compatibility with a range of ingress controllers and cloud-native routing mechanisms, including support for:
+
+* Wild card for Host based routing.
+* Creation of `httproutetemplates.rtf.mulesoft.com` templates to map a set of application endpoints to a set of arbitrary Http routing resources for Path based routing.
+* Synchronization of the domain or hosts from the Http Route templates, with Anypoint control plane.
+* Using parameter placeholders on `httproutetemplates.rtf.mulesoft.com` templates for additional extensibility.
```

Comment:
Also not parallel.

Reply:
Suggest using the generic "HTTP route templates" (instead of the code) for most mentions unless you're referring to a file or something you're manipulating.